code/frameworks/vue-vite/src/preset.ts

```diff
@@ -28,16 +29,12 @@ export const viteFinal: StorybookConfig['viteFinal'] = async (config, { presets
 
   plugins.push(vueDocgen());
 
-  const updated = {
-    ...config,
+  return mergeConfig(config, {
     plugins,
```

Comment:
Oh I see why this broke, plugins was replacing the original config, and now it's being merged.  I think now that we are merging, we should set `plugins: [vueDocgen()]` and remove the couple of lines above.